gramlib/grammar.ml

```diff
@@ -1663,7 +1668,9 @@ module Entry = struct
       edesc = Dlevels []}
   let create = make
   let parse (e : 'a t) p : 'a =
+    Control.check_for_interrupt ();
```

Comment:
What is this for?

Reply:
Oh, that leaked from my internal tree. This improves coq-lsp latency in quite a few cases, but doesn't belong here.